src/shared/components/post/post-listing.tsx

```diff
@@ -623,6 +624,141 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
     }
   }
 
+  duplicatesInfo() {
+    switch (this.props.showDupes) {
+      case "Small":
+        return this.smallDuplicates();
+      case "Expanded":
+        return this.expandedDuplicates();
+      case "DontRemove":
+        return <></>;
```

Comment:
`DontRemove` is confusing, seems like it should really be called `Hide`.

Reply:
Ya that'd be fine, I'll do that now.